src/vm/i386/excepcpu.h

```diff
@@ -28,12 +28,37 @@ class Thread;
                               // Actually, the handler getting set is properly registered
 #endif
 
+#ifdef FEATURE_PAL
+
+extern VOID SetSEHRecord(PEXCEPTION_REGISTRATION_RECORD record);
```

Comment:
Ok, I haven't realized that these functions are defined. Let's leave it as it is then.